index.html

```diff
@@ -72,6 +72,22 @@ <h2 class="promo-slogan">The right ingredients<br>for the right food</h2>
         </div>
       </section>
     </main>
+
+    <section class="about-section">
+      <section class="our-mission">
```

Comment:
Looks even like `figure` with `figcaption` and `blockquote`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure#quotations

```html
<section class="about-section">
  <figure>
    <figcaption>
      <h3 />
      <img />
    </figcaption>
    <blockquote />
    <img />
  </figure>
</section>
```